src/layouts/components/Header.tsx

```diff
@@ -10,14 +10,27 @@ import { Container } from "@components/commons/Container";
 import { SearchBar } from "@components/commons/searchbar/SearchBar";
 import { Menu, Transition } from "@headlessui/react";
 import { useWindowDimensions } from "hooks/useWindowDimensions";
-import { HeaderCountBar } from "./HeaderCountBar";
+import { useWhaleApiClient } from "@contexts/WhaleContext";
+import { WhaleApiClient } from "@defichain/whale-api-client";
+import {
+  ListProposalsType,
+  ListProposalsStatus,
+} from "@defichain/jellyfish-api-core/dist/category/governance";
+import { GovernanceProposalStatus } from "@defichain/whale-api-client/dist/api/governance";
 import { HeaderNetworkMenu } from "./HeaderNetworkMenu";
+import { HeaderCountBar } from "./HeaderCountBar";
 
 export function Header(): JSX.Element {
   const [menu, setMenu] = useState(false);
   const [atTop, setAtTop] = useState(true);
   const [isSearchIconClicked, setIsSearchIconClicked] = useState(false);
   const router = useRouter();
+  const [openProposals, setOpenProposals] = useState(0);
+  const api = useWhaleApiClient();
+
+  useEffect(() => {
+    getOpenProposals(api).then((res) => setOpenProposals(res));
+  }, []);
```

Comment:
let's update the number on a set of interval

Reply:
hmm since `layout/` will not reload on page navigation, let's add an interval, refer to `src/store/stats.tsx`